lib/gtfs/route.ex

```diff
@@ -15,10 +17,13 @@ defmodule Gtfs.Route do
     :id
   ]
 
-  @spec from_csv_row(%{required(String.t()) => String.t()}) :: t()
-  def from_csv_row(row) do
+  @spec from_csv_row(Csv.row()) :: t()
+  def from_csv_row(route_row) do
     %__MODULE__{
-      id: row["route_id"]
+      id: route_row["route_id"]
     }
   end
+
+  @spec bus_route?(Csv.row()) :: boolean
+  def bus_route?(route_row), do: route_row["route_type"] == "3"
```

Comment:
Sorry, I meant the function names could get a `_row` suffix, so `bus_route_row?`. This isn't a strong opinion. I'm just bringing it up again cuz the old comment got lost in the new commits.

Reply:
Oh, I misunderstood. Yeah, that makes sense.